### PR TITLE
[LDC] move `pragma(inline, true);` out of function bodies

### PR DESCRIPTION
**Pull request by thewilsonator**
cc @kinke

### CODE REVIEW

**Comment by Geod24** [comment]
Can you expand on the rationale ?

**Comment by thewilsonator** [comment]
see https://github.com/dlang/phobos/pull/7463#discussion_r502157278 and https://github.com/dlang/phobos/pull/7463#discussion_r502180674

**Comment by Geod24** [comment]
Thanks. A mention in the commit message would have been great, too, so people can get this information offline / without having to go through 2 indirections :)

File: std/math.d [hunk 1]
```diff
@@ -4038,11 +4048,9 @@ float scalbn(float x, int n) @safe pure nothrow @nogc { return _scalbn(x,n); }
     assert(scalbn(2.0,10) == 2048.0);
     assert(scalbn(2048.0f,-10) == 2.0f);
 }
-
+pragma(inline, true)
```
Comment (Geod24) [inline std/math.d:4051]: Please re-add the newline

**Review by Geod24 — APPROVED** [review]
LGTM. Just a minor formatting comment. Feel free to self-merge once fixed.

**Comment by Geod24** [comment]
@WalterBright : The LDC developers raise an important point. To allow for cross-module inlining, `pragma(inline)` should be outside of the function body, otherwise the compiler will have to analyze body of functions for non-root modules.
Should we consider deprecating usage of `pragma(inline)` inside of function body, and restrict it to be on the function's prototype ?

**Pull request merged**